src/test/ddms_tests.cpp

```diff
@@ -0,0 +1,233 @@
+#include <validation.h>
+#include <consensus/tx_verify.h>
+#include <consensus/validation.h>
+#include <index/txindex.h>
+#include <script/script.h>
+#include <test/test_bitcoin.h>
+#include <util/memory.h>
+
+#include <memory>
+
+#include <boost/test/unit_test.hpp>
+
+static CMutableTransaction createCoinbase() {
+	CMutableTransaction coinbaseTx;
+
+	coinbaseTx.nVersion = 1;
+	coinbaseTx.vin.resize(1);
+	coinbaseTx.vout.resize(1);
+	coinbaseTx.vin[0].scriptSig = CScript() << OP_11 << OP_EQUAL;
+	coinbaseTx.vin[0].prevout.SetNull();
+	coinbaseTx.vout[0].nValue = 50000;
+	coinbaseTx.vout[0].scriptPubKey = WDMO_SCRIPT;
+
+	return coinbaseTx;
+}
+
+static CMutableTransaction createLicenseTransaction(const uint256 parentHash) {
+	CMutableTransaction tx;
+	tx.vin.resize(1);
+	tx.vin[0].scriptSig = CScript() << OP_11;
+	tx.vin[0].prevout.hash = parentHash;
+	tx.vin[0].prevout.n = 0;
+	tx.vout.resize(2);
+	tx.vout[0].scriptPubKey = CScript();
+	tx.vout[0].nValue = 49000;
+	tx.vout[1].scriptPubKey = CScript() << OP_RETURN << std::vector<unsigned char>{3, 76, 84, 120, 20, 169, 20, 230, 53, 247, 106, 15, 189, 177, 112, 86, 88, 161, 227, 176, 86, 115, 120, 7, 74, 0, 5};
```

Comment:
Might be worthy to split the vector into logical parts (i.e. header, script_size, script, hashrate) and extract them into constants. Also, please specify the values in hex format.
One more thing - are you sure that the 5th value in the vector (20) is correct? It should specify the script length + 2, but the script itself is 23 bytes long.

Reply:
Done (structure of LTx CScript changed)